cli/cmd/verify.go

```diff
@@ -175,7 +175,7 @@ func parseVerifyFlags(cmd *cobra.Command) (*verifyFlags, error) {
 }
 
 func newCoordinatorValidateOptsGen(mnfst manifest.Manifest, hostData []byte) (*snp.StaticValidateOptsGenerator, error) {
-	validateOpts, err := mnfst.SNPValidateOpts()
+	validateOpts, err := mnfst.AKSValidateOpts()
```

Comment:
Should a manifest just have a `ValidateOpts` method that returns opts for all configured reference values?

Reply:
This would need to be generic then, which might get.. ugly?
I'd advocate for doing this once we _need_ the TDX validate opts at all.

Reply:
Fine with me.